specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-11-02-preview/scheduledEvents.json

```
@@ -0,0 +1,175 @@
+          },
+          {
+            "$ref": "./managedClusters.json#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ScheduledEventNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ScheduledEvent"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./managedClusters.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Scheduled Event": {
+            "$ref": "./examples/ScheduledEventsGet.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ScheduledEvent": {
+      "type": "object",
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The description of a scheduled event."
+        },
+        "eventId": {
+          "type": "string",
+          "description": "The event id of a scheduled event."
+        },
+        "eventSource": {
+          "type": "string",
+          "description": "The source of a scheduled event."
+        },
+        "eventStatus": {
+          "type": "string",
+          "enum": [
+            "Scheduled",
+            "Started",
+            "Completed",
+            "Cancelled",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "eventStatus",
+            "modelAsString": true
+          },
+          "description": "The status of a scheduled event."
+        },
+        "scheduledTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time of a scheduled event."
+        },
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time of a scheduled event starts."
+        },
+        "lastUpdateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time of a scheduled event last updates."
+        },
+        "resources": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "format": "arm-id",
+            "x-ms-arm-id-details": {
+              "allowedResources": [
+                {
+                  "type": "Microsoft.ContainerService/managedClusters"
+                },
+                {
+                  "type": "Microsoft.ContainerService/managedClusters/agentPools"
+                }
+              ]
+            }
+          },
+          "description": "The list of resources of a scheduled event."
+        },
+        "resourceType": {
+          "type": "string",
+          "enum": [
+            "ManagedCluster",
+            "AgentPool"
+          ],
+          "x-ms-enum": {
+            "name": "resourceType",
```

```suggestion
            "name": "ResourceType",
```

The name of the enum should be uppercase because it's a type.

---

Done.